cmd/server-main.go

```diff
@@ -82,13 +82,6 @@ var ServerFlags = []cli.Flag{
 		Usage:  "bind to a specific ADDRESS:PORT for embedded Console UI, ADDRESS can be an IP or hostname",
 		EnvVar: "MINIO_CONSOLE_ADDRESS",
 	},
-	cli.DurationFlag{
-		Name:   "shutdown-timeout",
-		Value:  xhttp.DefaultShutdownTimeout,
-		Usage:  "shutdown timeout to gracefully shutdown server",
-		EnvVar: "MINIO_SHUTDOWN_TIMEOUT",
-		Hidden: true,
-	},
```

Comment:
We need to keep this flag to not break deployments upon, upgrade, just that we don't have to honor the flags.